websub-ballerina/subscriber_service.bal

```diff
@@ -17,11 +17,12 @@
 # The WebSub service type
 public type SubscriberService service object {
 
-//  remote function onSubscriptionValidationDenied(SubscriptionDeniedError msg) returns Acknowledgement;
+//  remote function onSubscriptionValidationDenied(SubscriptionDeniedError msg) returns Acknowledgement?;
 
 //  remote function onSubscriptionVerification(SubscriptionVerification msg)
-//                    returns SubscriptionVerificationSuccess|SubscriptionVerificationError;
+//                    returns SubscriptionVerificationSuccess | SubscriptionVerificationError;
```

Comment:
Let's remove the spacing around the "|".

Reply:
* Removed unnecessary white-spaces.